library/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 // Based on eslint-config-react-app; https://github.com/facebookincubator/create-react-app/blob/cd3d04b71e91f533bdbdc3856775e1da81d445cf/packages/eslint-config-react-app/index.js
 
-{
+module.exports = {
```

Comment:
Renaming this file will break existing projects. It should be added to the 'important changes' in the doc.

And since we move to javascript style, you might want to change the json syntax into javascript syntax here as well. Otherwise eslint might break on it ;-)

Reply:
The JSON file contained comments